content/token22-on-chain.md

```diff
@@ -412,7 +459,9 @@ We just need to make sure the `user` account has signed the transaction and that
     pub user_stake_entry: Account<'info, StakeEntry>,
 ```
 
-The `user_stake_entry` account requires a few more constraints. We need to initialize the account, derive the address using the expected seeds, define who is paying for the creation of the account, and allocate enough space for the `StakeEntry` data struct. We deserialize the given account into the `StakeEntry` account.
+The `user_stake_token_account` is again the account where the user's staking rewards will eventually be sent. We create the account in this instruction so we don't have to worry about it later on when it's time to dole out the staking rewards. Because we initialize this account in this instruction, it puts a limit on the number of pools a user can stake in with the same reward token. This current design would prevent a user from creating another `user_stake_entry` account for another pool with the same `staking_token_mint`. This is another design choice that probably would not scale in production. Think about how else this could be designed.
+
+We use some similar SPL constraints as in the previous instruction, this time targeting the associated token program. With the `init` constraint, these tell anchor what mint, authority, and token program to use while initializing this associated token account.
```

Comment:
SPL constraints or Anchor constraints? I know they're constraints on SPL programs but this reads like SPL is constraining, rather than bering constrained.

Capital A for Anchor.